cmd/chain-maind/app/app.go

```diff
@@ -157,7 +157,7 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 			return err
 		}
 
-		return renameio.WriteFile(cleanedPath, bz, 0600)
+		return ioutil.WriteFile(cleanedPath, bz, 0600)
```

Comment:
We need to reopen this issue then. https://github.com/crypto-org-chain/chain-main/issues/398

Reply:
seems not possible on Windows: https://github.com/google/renameio#windows-support ?

Reply:
or keep using renameio, just pin the dependency to be before https://github.com/google/renameio/commit/8b2b09a2cfe0f9be5e3b41bdf85c0baf422c0f6f knowing it's not reliable on Windows?

Reply:
is it possible to conditional compile?

Reply:
it should be: https://golang.org/cmd/go/#hdr-Build_constraints

so I guess one other option is to separate it out and `// +build !windows` on "app.go"-subset and have some Windows "app.go"-subset ?

Reply:
@leejw51crypto

Reply:
ok, i'll apply conditional compile